java.j2seplatform/src/org/netbeans/modules/java/j2seplatform/resources/templates/defaultPlatform.xml

```diff
@@ -1,4 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
```

Comment:
This is a template. If this template is copied verbatim to the users code, the license should not be added. Please check that.

Reply:
I am not sure, this template only used in:
 java.j2seplatform\src\org\netbeans\modules\java\j2seplatform\resources\layer.xml

like this:

    <folder name="Services">
        <folder name="Platforms">
            <folder name="org-netbeans-api-java-Platform">
                 <!-- Placeholder, installed platforms will be created in this folder -->
                <file name="default_platform.xml" url="templates/defaultPlatform.xml">
                    <attr name="default-platform" boolvalue="true"/>
                </file>
            </folder>
        </folder>
    </folder>

Reply:
I had a look at this. Contrary to its name the file is not used as a template and is not shown to the user. Adding the header is correct in this case.